.github/CODEOWNERS

```diff
@@ -145,3 +145,19 @@ sycl/include/sycl/ext/oneapi/bindless* @intel/bindless-images-reviewers
 sycl/source/detail/bindless* @intel/bindless-images-reviewers
 sycl/plugins/unified_runtime/ur/adapters/**/image.* @intel/bindless-images-reviewers
 sycl/test-e2e/bindless_images/ @intel/bindless-images-reviewers
+
+# sycl e2e tests
+sycl/test-e2e/ @intel/llvm-reviewers-runtime
+sycl/test-e2e/GroupAlgorithm/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/GroupLocalMemory/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/SubGroup/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/SubGroupMask/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/Plugin/*level-zero* @intel/dpcpp-l0-pi-reviewers
```

Comment:
Now that all plugins are moved to a separate repo, and their ownership is outside of SYCL RT I suggest we completely remove the @intel/dpcpp-l0-pi-reviewers account:
```suggestion
```

Reply:
What I really mean is that the charter of @intel/dpcpp-l0-pi-reviewers (L0 plugin) is gone by now. Remaining PI and PI2UR functionality should be owned by @intel/llvm-reviewers-runtime. And eventually rewrite to completely remove PI and have SYCL RT talk directly to UR.

Reply:
There is a UR reviews team being setup , it would make sense to have them own the tests until they are fully moved per @aelovikov-intel suggestion. For now, leaving owners on this as is, and will update once new UR team reviewers team is setup (as dpcpp-l0-pi-reviewers as still the experts until UR officially moves it).